frontend/app/components/work-packages/wp-single-view/wp-single-view.directive.html

```diff
@@ -102,6 +102,6 @@ <h3 class="attributes-group--header-text" ng-bind="$ctrl.text.idLabel"></h3>
 
   <wp-attachments
       work-package="$ctrl.workPackage"
-      hide-empty-fields="$ctrl.hideEmptyFields">
+      data-ng-show="$ctrl.filesExist()">
```

Comment:
I worked on the drag and drop feature for so long that i forgot that one can actually drop files to the dropzone as well. so, you are right ;)